CMakeLists.txt

```diff
@@ -3,10 +3,15 @@
 # Distributed under the OSI-approved BSD 2-Clause License.  See accompanying
 # file `LICENSE` for more details.
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.12.1)
 
 project(freetype-gl LANGUAGES C CXX)
 
+set(LIBRARY_VERSION_MAJOR 1)
+set(LIBRARY_VERSION_MINOR 0)
+set(LIBRARY_VERSION_PATCH 0)
+set(LIBRARY_VERSION_STRING ${LIBRARY_VERSION_MAJOR}.${LIBRARY_VERSION_MINOR}.${LIBRARY_VERSION_PATCH})
```

Comment:
You're right we dot not have a library version and we can start with 1.0.0

Reply:
Please start with 0.1.0 if you want to introduce a versioning.  `1.0.0` implies `stable`.

Reply:
OK, I am struggling with RPM build right now in https://github.com/1div0/freetype-gl

Reply:
Just built 64-bit packages as follows:
rpmbuild -ba /usr/src/github.com/1div0/freetype-gl/packaging/rpm/freetype-gl.spec
rpmbuild --target=aarch64 -ba /usr/src/github.com/1div0/freetype-gl/packaging/rpm/freetype-gl.spec